src/lib/checkbox/checkbox.scss

```diff
@@ -470,6 +470,11 @@ md-checkbox {
 // Visually hidden but still able to respond to focus.
 .md-checkbox-input {
   @include md-visually-hidden;
+
+  // Move the input to the bottom and in the middle.
+  // Visual improvement to properly show browser popups when being required.
+  bottom: 0;
+  left: 50%;
```

Comment:
I was actually going to upload some screenshots, but this Safari thing distracted me too much.

> Screenshots are now in the PR description.

Also we already use an absolute position here, and I want the `50%` to be relative to the inner container.